packages/pages/src/vite-plugin/plugin.ts

```diff
@@ -17,8 +22,38 @@ const plugin = async (): Promise<PluginOption[]> => {
         process: "build",
       },
     }),
+    cleanup(projectStructure),
   ];
 };
 
+const clientHydration = async (
+  projectStructure: ProjectStructure
+): Promise<Plugin> => {
+  return {
+    name: "client-hydration:build",
+    apply: "build",
+    buildStart: {
+      sequential: true,
+      order: "pre",
+      async handler() {
+        await makeClientFiles(projectStructure);
+      },
+    },
+  };
+};
+
+const cleanup = async (projectStructure: ProjectStructure): Promise<Plugin> => {
+  return {
+    name: "client-hydration:cleanup",
+    apply: "build",
+    closeBundle: {
+      order: "post",
```

Comment:
Is this necessary?

Reply:
I don't believe so. No changes when testing so I'll remove